src/Compilers/Core/MSBuildTask/Portable/project.lock.json

```diff
@@ -468,6 +468,24 @@
           "lib/net45/_._": {}
         }
       },
+      "System.Collections.Immutable/1.1.38-beta-23428": {
```

Comment:
You should not be touching these files unless you've changed the project refererences.